### PR TITLE
June assets

### PR DESCRIPTION
**Pull request by morganen**

### CODE REVIEW

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Manually execute naming conventions on assets
+::: meta-data type="New" features="Asset Manager" available="late-June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
+:::
+
+In the Asset Manager, it is possible to define a naming convention in order to automatically populate asset attributes with values contained in the asset filename or asset code. This naming convention is executed at the asset creation.
+
+From now on, you can manually re-execute this naming convention afterward. It can be very useful if your naming convention changes after the asset upload/creation for instance.
```
Comment (felix-couprie) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:7]: "This naming convention was executed at the asset creation but from now on, you can also manually re-execute this naming convention afterward!"

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+You have 2 ways to manually execute the naming convention:
+- either in each asset edit form (it's a unitary action that saves the asset and execute the naming convention)
```
Comment (felix-couprie) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:10]: "and executes"

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Manually execute naming conventions on assets
+::: meta-data type="New" features="Asset Manager" available="late-June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
+:::
+
+In the Asset Manager, it is possible to define a naming convention in order to automatically populate asset attributes with values contained in the asset filename or asset code. This naming convention is executed at the asset creation.
```
Comment (NolwennP) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:5]: To rephrase. We need the new feature/improvement to be in the very first sentence as this is the one we will show in the communication panel of the PIM.

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+- either in each asset edit form (it's a unitary action that saves the asset and execute the naming convention)
+
+![Unitary execution](../img/unitary-execution.png)
+
+- or in the `product link rules` tab of your asset family (it's a mass action that executes the naming convention on all the assets of this asset family).
+
+![Mass execution](../img/mass-execution.png)
+
+::: more
+[About the product link rules](../articles/assets-product-link-rules.html)
```
Comment (NolwennP) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:19]: Each article link should be formulated as a question.

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+![Unitary execution](../img/unitary-execution.png)
+
+- or in the `product link rules` tab of your asset family (it's a mass action that executes the naming convention on all the assets of this asset family).
+
+![Mass execution](../img/mass-execution.png)
+
+::: more
+[About the product link rules](../articles/assets-product-link-rules.html)  
+[Create and display assets](../articles/create-and-display-assets.html)
```
Comment (NolwennP) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:20]: Each article link should be formulated as a question.

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+![Unitary execution](../img/unitary-execution.png)
+
+- or in the `product link rules` tab of your asset family (it's a mass action that executes the naming convention on all the assets of this asset family).
+
+![Mass execution](../img/mass-execution.png)
+
+::: more
+[About the product link rules](../articles/assets-product-link-rules.html)  
+[Create and display assets](../articles/create-and-display-assets.html)  
+[Manage your asset families](../articles/manage-asset-families.html)
```
Comment (NolwennP) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:21]: Each article link should be formulated as a question.

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Manually execute naming conventions on assets
+::: meta-data type="New" features="Asset Manager" available="late-June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
```
[the remark this answers is not in the document]
Comment (NolwennP) [reply content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:2]: It's either "early June", "mid-June" or "late June".

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+# Manually execute naming conventions on assets
+::: meta-data type="New" features="Asset Manager" available="late June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
+:::
+
+In the Asset Manager, it is possible to define a naming convention in order to automatically populate asset attributes with values contained in the asset filename or asset code. This naming convention is executed at the asset creation.
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:5]:
```suggestion
You can now manually execute naming conventions from the user interface whenever you need it. This new feature can be very useful if your naming convention has changed after asset uploads or creations.

Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation. 
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+From now on, you can manually re-execute this naming convention afterward. It can be very useful if your naming convention changes after the asset upload/creation for instance.
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:7]:
```suggestion
```
Moved at the beginning of the article

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+
+You have 2 ways to manually execute the naming convention:
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:9]:
```suggestion
You have two ways to manually execute a naming convention:
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+- either in each asset edit form (it's a unitary action that saves the asset and execute the naming convention)
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:10]:
```suggestion
- for one specific asset, directly in the asset edit form. The action is visible under the `...`
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+
+![Unitary execution](../img/unitary-execution.png)
+
+- or in the `product link rules` tab of your asset family (it's a mass action that executes the naming convention on all the assets of this asset family).
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:14]:
```suggestion
- for an asset family, under the `product link rules` tab of your asset family - the naming convention will be applied to all assets belonging to this family.
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+
+![Mass execution](../img/mass-execution.png)
+
+::: more
+[About the product link rules](../articles/assets-product-link-rules.html)
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:19]:
```suggestion
[What is a product link rule?](../articles/assets-product-link-rules.html)  
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+
+![Unitary execution](../img/unitary-execution.png)
+
+- or in the `product link rules` tab of your asset family (it's a mass action that executes the naming convention on all the assets of this asset family).
+
+![Mass execution](../img/mass-execution.png)
+
+::: more
+[About the product link rules](../articles/assets-product-link-rules.html)  
+[Create and display assets](../articles/create-and-display-assets.html)
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:20]:
```suggestion
[How to create and display assets?](../articles/create-and-display-assets.html)  
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+![Unitary execution](../img/unitary-execution.png)
+
+- or in the `product link rules` tab of your asset family (it's a mass action that executes the naming convention on all the assets of this asset family).
+
+![Mass execution](../img/mass-execution.png)
+
+::: more
+[About the product link rules](../articles/assets-product-link-rules.html)  
+[Create and display assets](../articles/create-and-display-assets.html)  
+[Manage your asset families](../articles/manage-asset-families.html)
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:21]:
```suggestion
[How to manage your asset families?](../articles/manage-asset-families.html)
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 16]
```diff
@@ -0,0 +1,22 @@
+# Manually execute naming conventions on assets
```
Comment (LaureBro) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:1]:
```suggestion
# New: execute asset naming conventions manually
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 17]
```diff
@@ -0,0 +1,23 @@
+# New: execute asset naming conventions manually
+::: meta-data type="New" features="Asset Manager" available="late June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
+:::
+
+You can now manually execute naming conventions from the user interface whenever you need it. This new feature can be very useful if your naming convention has changed after asset uploads or creations.
+
+Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation. 
+
+
+You have two ways to manually execute a naming convention:
```
Comment (camillebourjade) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:10]:
```suggestion
There are two ways to manually execute a naming convention:
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 18]
```diff
@@ -0,0 +1,23 @@
+# New: execute asset naming conventions manually
+::: meta-data type="New" features="Asset Manager" available="late June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
+:::
+
+You can now manually execute naming conventions from the user interface whenever you need it. This new feature can be very useful if your naming convention has changed after asset uploads or creations.
+
+Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation.
```
Comment (camillebourjade) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:7]:
```suggestion
Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation step. 
```

File: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md [hunk 19]
```diff
@@ -0,0 +1,23 @@
+# New: execute asset naming conventions manually
+::: meta-data type="New" features="Asset Manager" available="late June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
+:::
+
+You can now manually execute naming conventions from the user interface whenever you need it. This new feature can be very useful if your naming convention has changed after asset uploads or creations.
+
+Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation step.
```
Comment (NolwennP) [inline content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md:7]:
```suggestion
Asset naming conventions populate asset attributes with values contained in the asset filename or asset code. Prior to this improvement, naming conventions were executed once at the asset creation step. 
```

**Pull request merged**